packages/platforms/browser/lib/network-request-callback.ts

```diff
@@ -1,6 +1,6 @@
 export interface NetworkRequestInfo {
   url: string
-  readonly type: string
+  readonly type: 'fetch' | 'xmlhttprequest'
```

Comment:
if this is going to be used by the resource load plugin then it probably needs to be a more permissive type that corresponds to `PerformanceResourceTiming.initiatorType`  - it's only `RequestStartContext.type` that's restricted to `fetch`/`xmlhttprequest`